### ISSUE TITLE
contentctl is failing with creating new content.

### ISSUE THREAD

**Issue by yaroslav-nakonechnikov** [splunk/contentctl#154]
Hello, 

i've just cloned repository and in `main` branch did initial setup described there: https://github.com/splunk/contentctl?tab=readme-ov-file#from-source-advanced

then i tried to `init` it:
```
(contentctl-py3.11) user@pc:~/onedrive/git/contentctl$ contentctl init
Traceback (most recent call last):
  File "/mnt/c/Users/user/git/contentctl/contentctl/contentctl.py", line 196, in main
    init_func(t)
  File "/mnt/c/Users/user/git/contentctl/contentctl/contentctl.py", line 57, in init_func
    Initialize().execute(config)
  File "/mnt/c/Users/user/git/contentctl/contentctl/actions/initialize.py", line 23, in execute
    (config.path/emptyDir).mkdir(exist_ok=False)
  File "/usr/lib/python3.11/pathlib.py", line 1117, in mkdir
    os.mkdir(self, mode)
FileExistsError: [Errno 17] File exists: 'docs'
  File "/home/user/.cache/pypoetry/virtualenvs/contentctl-zVzUC8o4-py3.11/bin/contentctl", line 6, in <module>
    sys.exit(main())
  File "/mnt/c/Users/user/git/contentctl/contentctl/contentctl.py", line 225, in main
    traceback.print_stack()
```

it failed.  and tried several other times, till i've got that:
```
(contentctl-py3.11) user@pc:~/onedrive/git/contentctl$ mkdir APP
(contentctl-py3.11) user@pc:~/onedrive/git/contentctl$ contentctl init --app.title APP --app.label APP --path APP
The app 'APP' has been initialized. Please run 'contentctl new --type {detection,story}' to create new content
```

which is not expected, but understandable. 

later i've tried to create new content, and it is failing again:
```
(contentctl-py3.11) user@pc:~/onedrive/git/contentctl$ contentctl new --type detection --path APP
? enter detection name testing_detection
? what kind of detection is this endpoint
? enter author name yn
? enter author name yn
? select a detection type TTP
? Your data source done
? enter search (spl) _some_search_
? enter MITRE ATT&CK Technique IDs related to the detection, comma delimited for multiple T1003.003, T1003
? security_domain for detection endpoint
Traceback (most recent call last):
  File "/mnt/c/Users/user/git/contentctl/contentctl/contentctl.py", line 204, in main
    new_func(config)
  File "/mnt/c/Users/user/git/contentctl/contentctl/contentctl.py", line 92, in new_func
    NewContent().execute(config)
  File "/mnt/c/Users/user/git/contentctl/contentctl/actions/new_content.py", line 95, in execute
    YmlWriter.writeYmlFile(str(full_output_path), content_dict)
  File "/mnt/c/Users/user/git/contentctl/contentctl/output/yml_writer.py", line 10, in writeYmlFile
    with open(file_path, 'w') as outfile:
         ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'APP/detections/TTP/testing_detection.yml'
  File "/home/user/.cache/pypoetry/virtualenvs/contentctl-zVzUC8o4-py3.11/bin/contentctl", line 6, in <module>
    sys.exit(main())
  File "/mnt/c/Users/user/git/contentctl/contentctl/contentctl.py", line 225, in main
    traceback.print_stack()
```

why it asks for file which should be created by the `contenctl`?
as i see, it needs directory `APP/detections/TTP`, but why it doesn't create if it is not exist?

**Comment by ljstella** [comment]
Sorry for the delay in this! 

We've got an update in #162 that fixes both the new detection issue, as well as the analytic story category issue, and a few other small sharp edges. It should be merged shortly.

**Comment by ljstella** [comment]
#162 is now merged, and 4.0.5 is now released with fixes for these issues. Thanks for your patience!
